### PR TITLE
added-card-hover-project

### PR DESCRIPTION
**Pull request by murtazshaikh**
card-hover-effect

### CODE REVIEW

**Review by mujeeb100 — APPROVED** [review]

**Review by nehalskdev — CHANGES_REQUESTED** [review]
Why not ```css
*{
Margin: 0;
Padding:0;
Box-sizing:border-size;
}```

**Comment by murtazshaikh** [comment]
> Why not `css *{ Margin: 0; Padding:0; Box-sizing:border-size; }`

Okk I'll do changes.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+.container .card{
+    position: relative;
+}
+.container .card .face{
+    width: 300px;
+    height: 200px;
+    transition: 0.5s;
+}
+.container .card .face.face1{
```
Comment (syedabdullahali) [inline style.css:28]: Use Group similar  property @murtazshaikh

**Review by syedabdullahali — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    font-family: consolas;
+    min-height: 100vh;
+    background-color: #333;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.container{
+    width: 1000px;
+    position: relative;
```
Comment (syedabdullahali) [inline style.css:16]: Why position relative in container @murtazshaikh

**Review by syedabdullahali — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (murtazshaikh) [reply style.css:28]: done

File: style.css (hunk 2 above)
Comment (murtazshaikh) [reply style.css:16]: to place its child top left according to container.

**Pull request merged**